cogs/induct.py

```diff
@@ -193,12 +200,17 @@ async def _perform_induction(self, ctx: TeXBotApplicationContext, induction_memb
             )
             return
 
-        if induction_member.bot:
-            await self.command_send_error(
-                ctx,
-                message="Member cannot be inducted because they are a bot.",
-            )
-            return
+        intro_channel: discord.TextChannel | None = discord.utils.get(
+            self.bot.main_guild.text_channels, name="introductions",
```

Comment:
Function arguments should be separated onto individual lines for improved clarity